src/features/terms-and-privacy/provider.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import { TermsAndPrivacyModal } from './ui/TermsAndPrivacyModal';
+import { useShowUpdatePrompt } from './hooks';
+
+interface Props {
+  children: ReactNode;
+}
+
+export const TermsAndPrivacyProvider = ({ children }: Props) => {
+  const { showUpdatePrompt, hideUpdatePrompt } = useShowUpdatePrompt();
```

Comment:
`show` and `hide` in this context sounds similar. By reading this code I assume that I can do `showUpdatePrompt()` and `hideUpdatePromp()`. Is that true? :)

Maybe we can somehow improve this naming?

Reply:
Improved, thank you.
https://github.com/input-output-hk/nami/pull/911/commits/5c2fcd170239235d0c08c377a09261ebc86db884